README.md

```diff
@@ -11,211 +11,160 @@ to make it easy to setup monitoring for any env as well as to make it easy to mi
+| `MONITORING_TOOL__SERVICE_VERSION` | The service version, for example: `$CI_COMMIT_SHA`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |         ✔️         |          ✔️           |        ✔️        |
+| `MONITORING_TOOL__SERVICE_ENV`     | The service environment, for example: `$CI_ENVIRONMENT_NAME`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |         ✔️         |          ✔️           |        ✔️        |
+| `MONITORING_TOOL__CLIENT_TOKEN`    | This token is needed in order to send browser monitoring logs. You can create or find client token [here](https://app.datadoghq.com/organization-settings/client-tokens).                                                                                                                                                                                                                                                                                                                                                                                                                                                                |         ️          |                       |        ✔️        |
+| `MONITORING_TOOL__PUBLIC_PATH`     | This is RELATIVE path, part of URL between domain (which can be different for different environments) and path to file itself. In other words - base path for all the assets within your application.<br/>You can think of this as kind of relative [Public Path](https://webpack.js.org/guides/public-path/). For example it can be `/` or `/static`.<br/>In other words this is common relative prefix for all your static files or / if there is none.<br/> - for Vite.js the default is `/`<br/> - for Next.js the default is `/_next/static/chunks` (!!! `_` instead of `.` in file system)<br/> - for Gatsby.js the default is `/` |         ✔️         |                       |                  |
```

Comment:
now when you put it into table I think I have a confession to made. 
Last two env vars are actually params that should be passed to script. 
WDYT if we release new version where it is passed as named args? 
I think it will also simplify usage and this table.